hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -180,6 +183,13 @@ public void initialize(URI uri, Configuration configuration)
       blockOutputActiveBlocks = 1;
     }
 
+    if (configuration.getBoolean(FS_AZURE_ENABLE_BLOB_ENDPOINT, false)) {
```

Comment:
At this point abfsconfiguration variable is not created

Reply:
Only issue is that:
in future ...., in case we want to modify all locations where AbfsConfiguration.shouldEnableBlobEndPoint is called, we may not know that some code piece is using FS_AZURE_ENABLE_BLOB_ENDPOINT directly and may miss the change.

Reply:
its a nit; not pr merge stopper.